modules/material/material.graphql.js

```diff
@@ -3,6 +3,7 @@ const materialType = `
     _id: ID!
     name: [Language]
     description: [Language]
+    purpose: String
```

Comment:
What is `purpose`?

Reply:
A field to differentiate between "main" and "bottom" materials for the material page representation. We discussed it under the `type` name, but `type`, though not reserved, has a use in mongoose, so I'd rather avoid it

Reply:
I think, it is better to rename it to the `materialType`, because it reflects type

Reply:
No, it doesn't. It reflects the `purpose` for which the material is used in this context, it has nothing to do with the material's intrinsic characteristics